utils/test_results.py

```diff
@@ -296,11 +296,8 @@ def get_flake_aggregate_numbers(
         flakes = Flake.objects.filter(
             Q(repository_id=repo.repoid)
             & (
-                (Q(end_date__isnull=True))
-                | (
-                    Q(end_date__date__lte=until.date())
-                    & Q(end_date__date__gt=since.date())
-                )
+                Q(start_date__date__lte=until.date())
```

Comment:
start date < until date but end date > since date?

Could you remind me again what start and end mean in this context?
I'd figure we'd want start > until and end < since but I could just be messing with my head